app/code/Magento/Backend/Model/Menu/Director/Director.php

```diff
@@ -6,11 +6,16 @@
 
 namespace Magento\Backend\Model\Menu\Director;
 
+use Magento\Backend\Model\Menu\AbstractDirector;
+use Magento\Backend\Model\Menu\Builder;
+use Magento\Backend\Model\Menu\Builder\AbstractCommand;
+use Psr\Log\LoggerInterface;
+
 /**
  * @api
  * @since 100.0.2
  */
-class Director extends \Magento\Backend\Model\Menu\AbstractDirector
+class Director extends AbstractDirector
```

Comment:
Because we automatically merge changes from the upstream Magento Open Source repository, unnecessary changes like adding imports increase the possibility for future merge conflicts that have to be resolved manually.

I agree with the sentiment that imported classes are more readable, maybe if aliased nicely, but at this time keeping the code base close to upstream is also a consideration.

For this reason please remove the imports and use the full class names again, so the change can be merged more safely.